packages/charInfo_width/charInfo_width.1.1.0/opam

```diff
@@ -22,6 +22,6 @@ description: """
 This module is implemented purely in OCaml and the width function follows the prototype of POSIX's wcwidth."""
 
 url {
-  src:"https://bitbucket.org/zandoye/charinfo_width/get/1.1.0.tar.gz"
-  checksum: "md5=c4ab038e06f06a29692c05fdd7c268c5"
+  src:"https://github.com/kandu/charInfo_width/archive/1.1.0.tar.gz"
+  checksum: "md5=a539436d1da4aeb93711303f107bec7e"
```

Comment:
Why has the checksum changed? I think it would have been better to use the opam cache tarball and upload that. In any case, these were identical, the only difference between the tarballs is the lack of `.hg_archival.txt` and `.hgtags` so if somebody has comments we can point that out.